internal/oracles/mgr.go

```diff
@@ -91,30 +98,32 @@ func (omgr *OracleMgr) Start() {
 				omgr.logger.Info("Node is a validator and caught up with the network, starting oracles")
 
 				for name, start := range oracles.RegisteredOracles() {
-					go start(ctx2, &common.Service{
-						Logger:           omgr.logger.Named(name).Sugar(),
-						ExtensionConfigs: omgr.config,
-					}, &scopedKVEventStore{
-						ev: omgr.eventStore,
-						// we add a space to prevent collisions in the KV
-						// oracle names cannot have spaces
-						KV: omgr.eventStore.KV([]byte(name + " ")),
-					})
-					if err != nil {
-						omgr.logger.Error("failed to start oracle", zap.String("name", name), zap.Error(err))
-					}
+					go func(ctx context.Context, start oracles.OracleFunc, name string) {
```

Comment:
Possibly a mistake, this `ctx` parameter is not used in the closure, which captures the callers `ctx2` (which is fine).  Probably remove this input.